forte/data/data_store.py

```diff
@@ -283,7 +371,15 @@ def add_annotation_raw(self, type_name: str, begin: int, end: int) -> int:
         # annotation type entry data with default fields.
         # A reference to the entry should be store in both self.__elements and
         # self.__entry_dict.
-        raise NotImplementedError
+        entry = self._new_annotation(type_name, begin, end)
+        try:
+            self.__elements[type_name].add(entry)
+        except KeyError:
+            self.__elements[type_name] = SortedList(key=lambda s: (s[0], s[1]))
```

Comment:
So you have automatically creating the list here, I feel like there is no need to do this again in the `_add_new_type` function